### PR TITLE
Embracing GraphQL - How I migrated to Gatsby v1

### PR DESCRIPTION
**Pull request by timarney**
New post for review.

### CODE REVIEW

File: docs/blog/2017-09-26-embracing-graphql/index.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+
+## 1.0 Released
+
+When 1.0 landed, I waited for a while before trying to update my website.  The reviews we're solid but I was still stuck on my pre 1.0 mindset.
+
+Then on Sep 8 2017 I saw this tweet from @ryanflorence
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">.<a href="https://twitter.com/gatsbyjs">@gatsbyjs</a> is a completely original approach to web development. The mix of static/dynamic and how it handles data is really interesting.</p>&mdash; Ryan Florence (@ryanflorence) <a href="https://twitter.com/ryanflorence/status/906233888560758784">September 8, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
```
Comment (KyleAMathews) [inline docs/blog/2017-09-26-embracing-graphql/index.md]: This can be removed as we load the twitter script elsewhere.

Comment (timarney) [reply docs/blog/2017-09-26-embracing-graphql/index.md]: Just remove the script tag and keep the blockquote (embed)?

Comment (KyleAMathews) [reply docs/blog/2017-09-26-embracing-graphql/index.md]: Yup!

**Comment by KyleAMathews** [comment]
Awesome! Thanks so much for contributing this! Publishing now and will tweet out etc. from @gatsbyjs

**Comment by jlengstorf** [comment]
Hiya @timarney! 👋

This is definitely late, but on behalf of the entire Gatsby community, I wanted to say thank you for being here.

Gatsby is built by awesome people like you. Let us say “thanks” in two ways:

1.  **We’d like to send you some Gatsby swag.** As a token of our appreciation, you can go to the [Gatsby Swag Store][store] and log in with your GitHub account to get a coupon code good for one free piece of swag. (We’ve got t-shirts and hats, plus some socks that are really razzing our berries right now.)
2.  **If you’re not already part of it, we just invited you to join the Gatsby organization on GitHub.** This will add you to our team of maintainers. You’ll receive an email shortly asking you to confirm. By joining the team, you’ll be able to label issues, review pull requests, and merge approved pull requests.

If you have questions, please don’t hesitate to reach out to us: tweet at [@gatsbyjs][twitter] and we’ll come a-runnin’.

Thanks again! 💪💜

[store]: https://store.gatsbyjs.org
[twitter]: https://twitter.com/gatsbyjs

**Pull request merged**